src/preload.js

```diff
@@ -12,14 +12,18 @@ const notificationPath = `${app.getPath("userData")}/notification.jpg`;
 let currentSong = "";
 let player;
 let currentPlayStatus = statuses.paused;
+let barvalue;
+let updatecurrent = false;
```

Comment:
these variables need a better name :)

Reply:
Alright, you can suggest names if you have any already in mind.